Update information on setconfigfield and pageview (#5424)
segmentio/segment-docs: src/connections/destinations/catalog/actions-google-analytics-4-web/index.md

### Data is not sent to Google

> Ensure that the **setConfigurationFields** mapping is enabled and at least one other mapping has been configured and enabled in the destination mappings for an event you want to send to Google Analytics. If no mappings are enabled, the destination does not send events.

Reviewer: Are events sent if only the **setConfigurationFields** mapping is enabled? 

```suggestion
Ensure that you've enabled both the **setConfigurationFields** mapping and at least one other destination mapping for an event you want to send to Google Analytics. If no mappings are enabled, the destination does not send events.
```

### Differences between Google Analytics 4 Cloud and Google Analytics 4 Web destinations 

> Segment's [Google Analytics 4 Cloud](/docs/connections/destinations/catalog/actions-google-analytics-4/) server-side destination uses Google's Measurement Protocol API to send event data server to server, whereas Segment's [Google Analytics 4 Web](/docs/connections/destinations/catalog/actions-google-analytics-4-web/) device-mode destination loads the gtag.js library client-side and uses Segment's event data to map to gtag.js events directly. Each destination has its own advantages and disadvantages. Your choice between the two will depend on your specific use case, technical expertise, and the platforms from which you want to track data.

Reviewer: I know this is urgent - but in the future, is there a way you could expand further on what advantages and disadvantages exist?

```suggestion
Segment's [Google Analytics 4 Cloud](/docs/connections/destinations/catalog/actions-google-analytics-4/) server-side destination uses Google's Measurement Protocol API to send event data from server to server, while Segment's [Google Analytics 4 Web](/docs/connections/destinations/catalog/actions-google-analytics-4-web/) device-mode destination loads the gtag.js library client-side and directly maps Segment's event data to gtag.js events. Your choice between the two depends on your specific use case, technical expertise, and the platforms on which you want to track data.
```
